inc/c_util/two_d_array.h

```diff
@@ -10,15 +11,54 @@
 #endif
 
```

Comment:
This is not needed in this header, no usage of the PRI macros or the fixed size types.

Reply:
removed